src/router/index.ts

```diff
@@ -22,6 +22,11 @@ const routes: Array<RouteRecordRaw> = [
         path: 'dashboard',
         component: () => import('../pages/admin/dashboard/Dashboard.vue'),
       },
+      {
+        name: 'Payment methods',
+        path: '/pricing-plans',
+        component: () => import('../pages/pricing-plans/PricingPlans.vue'),
```

Comment:
Let's keep all names consistent (for route, navigation, component name). Please take name from figma.